pkg/wrapper/nmt_wrapper.go

```diff
@@ -89,7 +89,11 @@ func (w *ErasuredNamespacedMerkleTree) Push(data []byte) {
 // Root fulfills the rsmt.Tree interface by generating and returning the
 // underlying NamespaceMerkleTree Root.
 func (w *ErasuredNamespacedMerkleTree) Root() []byte {
-	return w.tree.Root()
+	root, err := w.tree.Root()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
lol we should fix this interface in rsmt2d as well

Reply:
+1 https://github.com/celestiaorg/rsmt2d/issues/156